app/views/application/equality-monitoring/disabilities.html

```diff
@@ -13,81 +13,75 @@
     {{ govukInput({
       classes: "govuk-!-width-two-thirds",
       label: {
-        text: "Describe your disability (optional)"
+        text: "Your disability or health condition (optional)"
       }
     } | decorateApplicationAttributes(["equality-monitoring", "disabilities-other"])) }}
   {% endset -%}
 
+  <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
+
+  <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
+
+  <!-- IF they haven’t put anything in the Ask for support if you’re disabled field -->
+  <p class="govuk-body">
+You can <a href="/application/{{ applicationId }}/additional-support" class="govuk-link">ask for support as part of your application</a> if you want to tell training providers about your needs straight away.    </p>
+  <!-- end if -->
+
+
   {{ govukCheckboxes({
     fieldset: {
+      legend: {
+        text: "Do you have any of these disabilities or health conditions?",
+        classes: "govuk-fieldset__legend--m"
+      },
       attributes: {
         "data-module": "clear-hidden"
       }
     },
     items: [{
-      text: "Blind",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "Or a serious visual impairment which is not corrected by glasses"
-      }
+      text: "Blindness or a visual impairment not corrected by glasses"
     }, {
-      text: "Deaf",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "Or a serious hearing impairment"
-      }
+      text: "Deafness or a serious hearing impairment"
     }, {
-      text: "Learning difficulty",
-      label: {
-        classes: "govuk-label--s"
-      },
-      hint: {
-        text: "For example, dyslexia, dyspraxia or ADHD"
-      }
+      text: "Dyslexia, dyspraxia or attention deficit hyperactivity disorder (ADHD)"
```

Comment:
We considered this. HESA's categories have an overlap here, as they have both of these:

>  ( ) Learning difference such as dyslexia, dyspraxia or AD(H)D
>  ( ) An impairment, health condition or learning difference not listed above

Which means that a learning difference other than dyslexia, dyspraxia or AD(H)D would fit both categories! To avoid this, we either need to remove learning difference from the "not listed above" category, or update the first label so that only dyslexia, dyspraxia and AD(H)D are listed.

Hard to know which to do - but the simpler approach I think is to have a single "not listed above" option which covers everything rather than having multiple "other" options. This would also let us see what people write in to the free text field, which will give us some idea of if there are any common options missing.

Reply:
We've decided to go with adding the "another learning difference" to the first option, and removing "learning difference" from the general "Another..." option:

> ( ) Dyslexia, dyspraxia or attention deficit hyperactivity disorder (ADHD) or another learning difference

and further down:

> ( ) Another disability, health condition or impairment affecting daily life

We can review this again in future. Thanks for your comment @edwardhorsford!